pyqt-apps/siriushla/as_ap_sofb/orbit_register.py

```diff
@@ -105,7 +105,7 @@ def get_registers_control(self):
 class OrbitRegister(QWidget):
     """Create the Context Menu for the Registers."""
 
-    DEFAULT_DIR = '/home/fac/sirius-iocs/si-ap-sofb'
+    DEFAULT_DIR = '/home/sirius/screens-iocs'
```

Comment:
The official (managed by ansible) path is `/home/sirius/mounts/screens-iocs`

Reply:
I fixed it @xresende, thanks!